### PR TITLE
enhance: update milvus.yaml

### PR DESCRIPTION
**Pull request by tedxu**
See #32168

### CODE REVIEW

File: configs/milvus.yaml [hunk 1]
```diff
@@ -200,132 +205,135 @@ proxy:
...
     serverMaxSendSize: 268435456
     serverMaxRecvSize: 67108864
     clientMaxSendSize: 268435456
     clientMaxRecvSize: 67108864
-  # query whose executed time exceeds the `slowQuerySpanInSeconds` can be considered slow, in seconds.
-  slowQuerySpanInSeconds: 5
 
 # Related configuration of queryCoord, used to manage topology and load balancing for the query nodes, and handoff from growing segments to sealed segments.
 queryCoord:
+  taskMergeCap: 1
```
Comment (yiwen92) [inline configs/milvus.yaml:254]: Any description or comments here?

Comment (tedxu) [reply configs/milvus.yaml:254]: Will fill the descriptions in future PRs, thanks!

File: configs/milvus.yaml [hunk 2]
```diff
@@ -144,54 +152,51 @@ rocksmq:
...
-  compressionTypes: [0, 0, 7, 7, 7]
+  compressionTypes: 0,0,7,7,7 # compaction compression type, only support use 0,7. 0 means not compress, 7 will use zstd. Length of types means num of rocksdb level.
 
 # natsmq configuration.
 # more detail: https://docs.nats.io/running-a-nats-service/configuration
 natsmq:
-  server: # server side configuration for natsmq.
-    port: 4222 # 4222 by default, Port for nats server listening.
-    storeDir: /var/lib/milvus/nats # /var/lib/milvus/nats by default, directory to use for JetStream storage of nats.
-    maxFileStore: 17179869184 # (B) 16GB by default, Maximum size of the 'file' storage.
```
Comment (yiwen92) [inline configs/milvus.yaml:158]: Why we choose to delete default value&unit in these params?

Comment (yiwen92) [reply configs/milvus.yaml:158]: OK. This is generated by code, code needs to add units&default value in future

**Pull request merged**